lib/steep/services/completion_provider.rb

```diff
@@ -364,6 +384,18 @@ def items_for_trigger(position:)
           # foo::← ba
           items.push(*items_for_colon2(position: position))
 
+        when node.type == :csend && at_end?(position, of: (_ = node.loc).dot) && (_ = node.loc).dot.source == "&."
```

Comment:
Seems like the last `source` check is unnecessary because `:csend` always has `&.` part, unlike `.` in `:send` nodes. (`1 + 2`)